docs/reference/commandline/swarm_init.md

```diff
@@ -64,7 +64,7 @@ You can disable autolock by running `docker swarm update --autolock=false`.
 After disabling it, the encryption key is no longer required to start the
 manager, and it will start up on its own without user intervention.
 
-### <a name=""></a> Configure node healthcheck frequency (--dispatcher-heartbeat)
+### <a name="configure-node-healthcheck-frequency---dispatcher-heartbeat"></a> Configure node healthcheck frequency (--dispatcher-heartbeat)
```

Comment:
Thanks! I think the name here needs to be `dispatcher-heartbeat` to match the name of the flag the section is for; that way Hugo will add a link from the table that lists all flags